test/stateUtilsTest.js

```diff
@@ -0,0 +1,182 @@
+import StateUtils from '../src/Application/Redux/StateUtils';
+
+import chai from 'chai';
+const assert = chai.assert;
+const expect = chai.expect;
+
+// let assert = require('assert');
+// let mocha = require('mocha');
+
+const modifiedState = StateUtils.getInitialState();
+modifiedState.money.cash++;
+modifiedState.money.total++;
+modifiedState.assets = StateUtils.createAssetsAfterBuy(modifiedState.assets, 2);
+
+
+// End Setup
+
+// Tests
```

Comment:
i'd just skip the `// End Setup` and `// Tests` thing.  in fact i'd put most setup in `beforeEach` at the appropriate describe level so you know no previous step fucking with it